.storybook/preview-head.html

```diff
@@ -1,31 +0,0 @@
-<script>
```

Comment:
I could not determine any difference in the color mode flicker with or without this file, so I "Marie Kondo"'d it.

Reply:
Same here. I removed it in my PR as well. So it's fine.